supabase/tests/64_test_canceled_org_version_cleanup.sql

```
@@ -0,0 +1,405 @@
+    FROM public.stripe_info
+    WHERE customer_id = (SELECT paying_customer FROM canceled_cleanup_ctx)
+  ),
+  'succeeded',
+  'process_free_trial_expired leaves paying orgs unchanged'
+);
+
+-- Full canceled-org cleanup (versions + audit logs)
+SELECT public.cleanup_long_canceled_org_data();
+
+SELECT is(
+  (SELECT deleted FROM public.app_versions WHERE id = 970101),
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)